packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js

```diff
@@ -96,7 +90,7 @@ export default /* @ngInject */ ($stateProvider) => {
       goToResiliation: /* @ngInject */ ($state) => () =>
         $state.href('vps.detail.dashboard.resiliation'),
       shouldReengage: /* @ngInject */ (vps) => vps.shouldReengage,
-
+      user: /* @ngInject */ (OvhApiMe) => OvhApiMe.v6().get().$promise,
```

Comment:
`user` no longer needed ?

Reply:
It is retained

Reply:
Could we use `coreConfig.getUser()` instead ? We avoid an API call that way (through ovh-api-services on top of that 😄